File: src/apps/Andriod/app/src/main/java/com/dkellycollins/triggerfinger/data/config/IBulletConfig.java
package com.dkellycollins.triggerfinger.data.config;

import com.dkellycollins.triggerfinger.data.model.IVector;

/**
 * Created by Devin on 9/7/2015.
 */
public interface IBulletConfig {

    float getCollidableRadius();

    IVector getVelocity();

    int getRespawnRate();
}

File: src/apps/Android-Kotlin/app/src/main/kotlin/com/dkellycollins/triggerfinger/data/config/IBulletConfig.kt
package com.dkellycollins.triggerfinger.data.config

import com.dkellycollins.triggerfinger.data.model.IVector

/**
 * Created by Devin on 9/7/2015.
 */
interface IBulletConfig {

    val collidableRadius: Float

    val velocity: IVector

    val respawnRate: Int

    val bitmapId: Int
}
